client/store/index.js

```diff
@@ -8,19 +8,18 @@ import categoryReducer from './category';
 import singleProductReducer from './singleProduct';
 import userOrdersReducer from './userOrders';
 import cartReducer from './cart';
-import authForm from './authForm';
+
 
 
 
 const reducer = combineReducers({
   auth,
-  authForm,
   category: categoryReducer,
   products,
   singleProduct: singleProductReducer,
   orders: userOrdersReducer,
   cart: cartReducer,
-  user: userAccountReducer
```

Comment:
Do you know why this was removed? This is needed for to update the user account details.